spec/test-cases/index.ts

```diff
@@ -47,15 +47,12 @@ export function getDataTypeUseCases(
    * Filters out exclusions and remaps into an array of arrays of sample data
    */
   return Object.keys(testCaseMap)
-    .reduce<any[][]>(
-      (acc, k) =>
-        testCaseMap.hasOwnProperty(k) && typeof k === 'number' &&
-        k in DataType && exclude.indexOf(k) === -1
-          ? acc.concat(testCaseMap[k])
-          : acc,
+    .reduce<any[]>(
+      (acc, k) => testCaseMap.hasOwnProperty(k) && k in DataType && exclude.indexOf(k) === -1
+        ? acc.concat(testCaseMap[<any>k])
+        : acc,
       []
-    )
-    .reduce<any[]>((a, b) => a.concat(b), []);
+    );
```

Comment:
Note: AVA was nice enough to surface a bug for me in the tests 👍 This gets fixed with this PR.